### ISSUE TITLE
Chance to keep MaterialV2?

### ISSUE THREAD

**Issue by Der-Flip** [zhanghai/MaterialFiles#1007]
I just read in the 1.6.0 changelog the MaterialV2 design theme will be retired with 1.7.0

Apart from features I really liked that Material Files still offered MaterialV2 as I genuinely dislike MaterialV3. It is overly clunky, wastes tonnes of space, and the combination of spherical and "rounded corner" elements.... Anyways

Is there a chance to have MaterialV2 - if not as the primary, default design - as the optional opt-in design? Basically the way it is now but in reverse?

I know designs change and usually for the better and I probably look like the boomer who can't accept things change but I just don't see what's improved in usability or UI-design in V3 over V2.

To me Material Files is really quite perfect the way it is now.

**Comment by zhanghai** [comment]
Personally I used to love Material 1 and genuinely hate Material 2 (those bottom tabs! meh). And I used to love Material 2 and hate Material 3. The thing is, once my system gets upgraded and my frequently used apps get upgraded to the new visuals, I feel it's more important to have consistency than sticking to certain now outdated design. So in the end the vast majority of people will switch to the new design, and Materail 3 has been out long enough for that.

The realistic reason to drop Material 2 is that I found myself spending more time than I expected to come up with 2 designs that work well with both Materail 2 and Material 3. My time that can be devoted to this project is limited so I need to make trade-offs on what I can maintain, and right now I feel Material 2 is no longer something that I feel worth spending my limited time on.

But rest assured, for the same reason about limited time, 1.7.0 won't come very soon (perhaps half a year later) and I may still release small new features with 1.6.x. It's just that when I need to make bigger (UI) changes, I will no longer write new things for Material 2 and will remove it instead.

(And of course, you are always free to maintain a Material 2 fork when 1.7.0 happens, which is what's good about being open-source.)

**Comment by sbot50** [comment]
I agree with this post, please dont remove material design 2 (which i assume is the default one without the mat design 3 toggle on), mat design 3 looks bad

**Comment by daiaji** [comment]
> Personally I used to love Material 1 and genuinely hate Material 2 (those bottom tabs! meh). And I used to love Material 2 and hate Material 3. The thing is, once my system gets upgraded and my frequently used apps get upgraded to the new visuals, I feel it's more important to have consistency than sticking to certain now outdated design in a fraction of apps. So in the end the vast majority of people will switch to the new design, and Materail 3 has been out long enough for that.
> 
> The realistic reason to drop Material 2 is that I found myself spending more time than I expected to come up with 2 designs that work well with both Materail 2 and Material 3 when I worked on 1.6.0. My time that can be devoted to this project is limited so I need to make trade-offs on what I can maintain, and right now I feel Material 2 is no longer something that I feel worth spending my limited time on.
> 
> But rest assured, for the same reason about limited time, 1.7.0 won't come very soon (perhaps half a year later) and I may still release small new features with 1.6.x. It's just that when I need to make bigger (UI) changes, I will no longer write new things for Material 2 and will remove it instead.
> 
> (And of course, you are always free to maintain a Material 2 fork when 1.7.0 happens, if you still feel strongly about it at that time, which is what's good about being open-source.)

MD1算是Google自己设计的吧，MD2不是那个iOS设计师弄出来吗？（经典底栏）
记得MD1的逻辑就是切换Tab就使用滑动，我认为这个逻辑还挺正常的。

**Comment by Secret-chest** [comment]
Please maintain Material 2 for as long as you can.
